docs/UserGuide.md

```diff
@@ -131,19 +143,35 @@ Examples:
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
 
+### Locating persons by phone number: `findnum`
+
+Finds persons whose phone number matches the given keywords.
+
+Format: `find PHONE_NUMBER`
+
+* The search is strictly numeric. e.g `83647382` will only match `83647382`.
+* The order of the keywords matters. e.g. `83647382` will not match `83467328`
+* Only the full number is searched. eg. `83647382` will not match `83641001`
+
+Examples:
+* `find 83647382` returns `john` and `John Doe`
+* `find 91739272` returns `David Li`<br>
+
 ### Deleting a person : `delete`
 
 Deletes the specified person from the address book.
 
-Format: `delete INDEX`
+Format: `delete [n/NAME] i/ID`
 
-* Deletes the person at the specified `INDEX`.
+* Deletes the person with the specified `ID`.
 * The index refers to the index number shown in the displayed person list.
 * The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+* `list` followed by `delete i/2` deletes the person with an ID of 2 in the address book.
+
+[//]: # (requires confirmation:)
+[//]: # (* `find Betsy` followed by `delete 1` deletes the person with ID of 1 in the results of the `find` command.)
```

Comment:
Um... actually not sure.. I think we previously decided that id refers to the unique id attached to each person instead of the order of persons in the displayed list. Which raises the question of why we don't allow operations using the id as in the order of persons in the displayed list. TBD